src/dmd/dmangle.d

```diff
@@ -510,7 +510,7 @@ public:
 
         if (ta.purity)
             buf.writestring("Na");
-        if (ta.isnothrow)
+        if (ta.throw_ == THROW.nothrow_)
```

Comment:
I think you need to add a mangling rule for `THROW.throw_` and tests for `__traits(getFunctionAttributes, ...)`.

Reply:
What do you mean? There is no need to add new mangling for `throw`. The same as trust, only `safe` and `trusted` has mangling rules, `system` is considered the default, and therefore no mangling rule is needed.